src/config/localfile-config.c

```diff
@@ -99,6 +101,8 @@ int Read_Localfile(XML_NODE node, void *d1, __attribute__((unused)) void *d2)
     logf[pl].exists = 1;
     logf[pl].future = 1;
     logf[pl].reconnect_time = DEFAULT_EVENTCHANNEL_REC_TIME;
+    logf[pl].regex_ignore = NULL;
```

Comment:
Once read both expressions, can we check that the same expression was not defined for both tags?
```
<localfile>
  <ignore>\d+.\d+.\d+.\d+</ignore>
  <restrict>\d+.\d+.\d+.\d+</restrict>
</localfile>
```

Logs that match that expression will be ignored and also restricted so no logs will be read for that localfile. This should be an invalid configuration.